proto_parser/src/proto_parser.h

```diff
@@ -30,10 +30,20 @@ class ProtoParser {
 
   ProtoParser() {}
 
-  ProtoParser(absl::string_view text_proto);
+  ProtoParser(absl::string_view text_proto) {
+    lines_ = absl::StrSplit(text_proto, '\n');
+  }
+
+  struct PrevFieldLine {
+
+    PrevFieldLine(int line_in) 
+      : line(line_in) {}
```

Comment:
I don't think you need the constructor.

Reply:
Should I remove it then? I just made it so that whenever I create this struct I can use one line instead of two